### ISSUE TITLE
https fech package at go.opencensus.io times out with wrong import format 

### ISSUE THREAD

**Issue by adiack** [census-instrumentation/opencensus-go#953]
**Describe the bug**
not able to fetch packages via 'go get -u' 
Mac OS  10.13.6 

**To Reproduce**

go get -u go.opencensus.io
package go.opencensus.io: unrecognized import path "go.opencensus.io" (https fetch: Get https://go.opencensus.io?go-get=1: dial tcp 216.239.32.21:443: i/o timeout)

**Expected behavior**
Package installed.

**Additional context**
similar to issue https://github.com/grpc/grpc-go/issues/1959

Workaround was for me to get whole repository.
git clone https://github.com/census-instrumentation/opencensus-go
then rename src/opencensus-go to src/go.opencensus.io

Am sure there is a better way to this though?

**Comment by ramonza** [comment]
Seems to be working now, perhaps this was a temporary network issue or issue AppEngine. Hopefully we will be able to improve this experience in time through the use of caching with the new Go module system.

**Comment by kochurovro** [comment]
Hi @ramonza  Can u check this problem again? I have it

go: go.opencensus.io@v0.18.0: unrecognized import path "go.opencensus.io" (https fetch: Get https://go.opencensus.io?go-get=1: dial tcp 216.239.32.21:443: i/o timeout)

**Comment by athul20** [comment]
Hi,
 I wanted to install cloudprober. So I installed go and tried installing cloudprober, it showed   cannot find package "go.opencensus.io/plugin/ocgrpc",  then I tried installing opencensus using go get -u go.opencensus.io its showing package go.opencensus.io: unrecognized import path "go.opencensus.io" (https fetch: Get https://go.opencensus.io?go-get=1: dial tcp: lookup go.opencensus.io on 10.228.0.2:53: no such host).

Can someone tell me how to solve the issue?
